modules/crud.js

```diff
@@ -0,0 +1,73 @@
+// import { todoListss } from '../src/index.js';
+
+// logic to create new todo using javascript class
+// export default class NewTodo {
+//   constructor(description) {
+//     this.description = description;
+//     this.id = todoListss.length + 1;
+//   }
+
+//   add = () => {
+//     const item = {
+//       id: this.id,
+//       description: this.description,
+//       completed: false,
+//       disabled: true,
+//       icon: 'more_vert',
+//     };
+//     createTodo(item);
+//     todoListss.push(item);
+//   };
+// }
```

Comment:
- [ ] As requested by my previous colleague, you need to remove commented code because it makes your work not clean and unprofessional.